README.md

```diff
@@ -176,505 +85,8 @@ will also be happy to provide advice or receive feedback.
 
 ## Usage
 
-```go
```

Comment:
How about linking one or several examples from the README?

Reply:
I don't like it. Everything in a module documentation, and I say it in a README.

Reply:
I have looked over several go packages and it's not rare for them to contain at least a minimal example.

https://github.com/natefinch/lumberjack
https://github.com/disintegration/imaging

Reply:
Who will test the example in a README?

Reply:
> Now the Usage section looks so:
> It does not describe usage of the connector anymore. Now it looks as an orphan piece of information.

Remove section at all and moving wording about uuid module to uuid example.